Bento/Components/Button.swift

```diff
@@ -182,7 +182,7 @@ extension Component.Button {
 }
 
 public extension Component.Button {
-    public final class StyleSheet: InteractiveViewStyleSheet<View> {
+    final class StyleSheet: InteractiveViewStyleSheet<View> {
         public let button: ButtonStyleSheet
```

Comment:
These `public`s will be confusing, so it's better to also remove all of them.

Reply:
I just removed the ones that were causing compiler warnings.

Reply:
Never mind, it's `public extension` which is already confusing... 🙄